CONTRIBUTING.md

```
@@ -36,23 +36,21 @@ Have ideas for new features or use cases? We're eager to hear them! But first:
 
 - **1.2. Create a branch**: create the branch that you will work on by using the link provided in the issue details page (right panel at the bottom - section "Development")
 
-- **1.3. Setup the SDK locally**: clone the repository
+- **1.3. Setup the SDK locally**: clone the repository and install dependencies
 
    ```sh
    $ git clone https://github.com/genlayerlabs/genlayer-py.git
```

markdownlint-cli2 (0.21.0)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)